### DISCUSSION TITLE
createrepo in brew

### DISCUSSION THREAD

**Discussion by ep4sh** [Homebrew/discussions#2897] [Casks] [answered]
Dear all,

I'd appreciate it if somebody could tell me if there is "createrepo" (create metadata for RPM repositories) application for macOS?
git clone http://createrepo.baseurl.org/git/createrepo.git

I wonder how can I help to port it, in case the package is missing?

Cheerz,
Pasha

**Comment by SMillerDev** [comment, marked as answer]
https://formulae.brew.sh/formula/ says no and so does https://repology.org/project/createrepo/versions

**Reply by ep4sh** [reply]
@SMillerDev 
Thank you, Sean, could you please elaborate more on how can I create a brew package with this app?

**Reply by carlocab** [reply]
Please see https://docs.brew.sh/Adding-Software-to-Homebrew.

**Reply by viking1304** [reply]
@ep4sh you will probably just waste your time, since web certificates are not OK.

If you open that url over http, you will get this error:

```
Not Found
The requested URL /git/createrepo.git was not found on this server.

Apache Server at createrepo.baseurl.org Port 80
```

If you open that url over https, you will get this message:

```
Your connection is not private
Attackers might be trying to steal your information from createrepo.baseurl.org (for example, passwords, messages, or credit cards). Learn more
NET::ERR_CERT_COMMON_NAME_INVALID
```

It use free Let's Encrypt certificate, but it is configured for other domain, not for that one.

I have never seen any Homebrew formula that ignores invalid certificates.

Is there any link with proper certificates?

**Reply by ep4sh** [reply]
@carlocab thank you for sharing!

@viking1304 I guess the only few URLs I've found are:
- Project: http://createrepo.baseurl.org/
- Download: http://createrepo.baseurl.org/download/index.html

**Reply by viking1304** [reply]
> * Project: http://createrepo.baseurl.org/
> * Download: http://createrepo.baseurl.org/download/index.html

Ouch!

Project page states that lates stable version is 0.4.11, and that version is from 26-Nov-2007! Almost 15 years old release is highly unlikely candidate for official Homebrew repo. You still might want to try to make unofficial formula and publish it on your own github.

You might want to try this active PyPI package instead https://pypi.org/project/createrepo-c/ If that works for you, you do not need to try to prepare formula for an outdated project.

**Reply by ep4sh** [reply]
Thank you all, guys!
